app/javascript/mastodon/components/short_number.tsx

```diff
@@ -0,0 +1,111 @@
+import { memo } from 'react';
+
+import { FormattedMessage, FormattedNumber } from 'react-intl';
+
+import { toShortNumber, pluralReady, DECIMAL_UNITS } from '../utils/numbers';
+
+/**
+ * Custom renderer for numbers
+ * @param {JSX.Element} displayNumber Number to display
+ * @param {number} pluralReady Number used for pluralization
+ * @returns {JSX.Element} Final render of number
+ */
+interface ShortNumberRenderer {
```

Comment:
I would prefer function definitions to be written as types rather than interfaces:

```ts
type ShortNumberRenderer = (displayNumber: JSX.Element, pluralReady: number) => JSX.Element;
```

Reply:
I had assumed that the policy was to align with interface.
I too think type is easier to read.